--- lsdvd-0.16/lsdvd.c.orig	2006-03-02 14:48:11.000000000 +0100
+++ lsdvd-0.16/lsdvd.c	2006-03-04 21:17:44.363323500 +0100
@@ -13,13 +13,13 @@
  *  2003-04-19  Cleanups get_title_name, added dvdtime2msec, added helper macros,
  *			  output info structures in form of a Perl module, by Henk Vergonet.
  */
-#include <dvdread/ifo_read.h>
 #include <string.h>
 #include <sys/stat.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
 #include <stdint.h>
+#include <dvdread/ifo_read.h>
 #include "lsdvd.h"
 #include "ocode.h"
 
--- lsdvd-0.16/configure.in.orig	2005-12-12 21:11:01.000000000 +0100
+++ lsdvd-0.16/configure.in	2006-03-04 21:18:12.821102000 +0100
@@ -1,7 +1,7 @@
 dnl Process this file with autoconf to produce a configure script.
 AC_INIT(lsdvd.c)
 AM_INIT_AUTOMAKE(lsdvd, 0.16)
-CFLAGS="$CFLAGS -g -D_FILE_OFFSET_BITS=64"
+CFLAGS="$CFLAGS -D_FILE_OFFSET_BITS=64"
 AC_PROG_CC
 
 if eval "test x$GCC = xyes"; then # Enable warnings when using GCC
@@ -10,8 +10,8 @@
 
 AC_CHECK_LIB(dvdread, DVDOpen, , AC_MSG_ERROR([libdvdread not found!]))
 AC_MSG_CHECKING([for dvdread/ifo_read.h])
-AC_TRY_COMPILE([#include <dvdread/ifo_read.h>
-		#include <stdint.h>], ,
+AC_TRY_COMPILE([#include <stdint.h>
+		#include <dvdread/ifo_read.h>], ,
 	AC_MSG_RESULT([yes]), 
 	AC_MSG_RESULT([no]) 
 	AC_MSG_ERROR([Header files for dvdread not found]))
